Symfony/CS/Fixer/Contrib/NoUselessReturnFixer.php

```diff
@@ -66,7 +66,10 @@ public function getPriority()
     private function fixFunction(Tokens $tokens, $start, $end)
     {
         for ($index = $end; $index > $start; --$index) {
-            if (!$tokens[$index]->isGivenKind(T_RETURN)) {
+            if (
+                !$tokens[$index]->isGivenKind(T_RETURN) ||
+                $tokens[$tokens->getPrevMeaningfulToken($index)]->isGivenKind(T_ELSE)
```

Comment:
what about just

```
if ($foo)
    return;
```

removing that will break stuff if it's at the end of a method